### PR TITLE
placeholder bug fixed

### PR DESCRIPTION
**Pull request by aryankoul**
@RocketChat/ReactNative

Input in upload modal was displaying full text instead of hint text

### CODE REVIEW

**Comment by aryankoul** [comment]
> Thanks!
> If the user submit a file upload without a filename, it'll raise an error though.
> Can you prevent him to do it? It's related to the placeholder bug anyway.

@diegolmello i have put a check for filename in upload model

File: app/containers/MessageBox/UploadModal.js [hunk 1]
```diff
@@ -119,6 +119,10 @@ export default class UploadModal extends Component {
 	submit = () => {
 		const { file, submit } = this.props;
 		const { name, description } = this.state;
+		if (name === '') {
+			ToastAndroid.show('Please enter a filename', ToastAndroid.SHORT);
```
Comment (diegolmello) [inline app/containers/MessageBox/UploadModal.js]: What about iOS?

Comment (aryankoul) [reply app/containers/MessageBox/UploadModal.js]: @diegolmello i have changes Toast to alert.

File: app/containers/MessageBox/UploadModal.js [hunk 2]
```diff
@@ -119,6 +119,10 @@ export default class UploadModal extends Component {
 	submit = () => {
 		const { file, submit } = this.props;
 		const { name, description } = this.state;
+		if (name === '') {
+			Alert.alert('Oops!', 'Filename is required.');
```
Comment (diegolmello) [inline app/containers/MessageBox/UploadModal.js]: Use I18n

File: app/containers/MessageBox/UploadModal.js [hunk 3]
```diff
@@ -119,6 +119,10 @@ export default class UploadModal extends Component {
 	submit = () => {
 		const { file, submit } = this.props;
 		const { name, description } = this.state;
+		if (name === '') {
```
Comment (diegolmello) [inline app/containers/MessageBox/UploadModal.js]:
```jsx
if (!name) {
```

**Pull request closed without merging**